src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java

```diff
@@ -0,0 +1,95 @@
+package com.st.utopia.cancellationservice.model;
+
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+/**
+ * A class to be the primary key of a {@link Ticket} for JPA, which requires every
+ * Entity to have a single primary key.
+ *
+ * <p>Declared to implement Serializable because "Composite-id class must implement Serializable."
+ *
+ * @author Al-amine AHMED MOUSSA
+ */
+
+@Embeddable
+public class TicketIdentity implements Serializable {
+	/**
+	 * Serialization version. Increment on any change to class structure that is
+	 * pushed to production.
+	 */
+	private static final long serialVersionUID = 1L;
+	/**
+	 * the flight ID.
+	 */
+	@Column(name="flight")
+	private Integer flight;
+	/**
+	 * the row of the seat .
+	 */
+	@Column(name="seat_row")
+	private Integer row;
+	/**
+	 * the seat group.
+	 */
+	@Column(name="seat")
+	private Character seat;
+	/**
+	 * the Getters and Setters of the Class
+	 */
+	public Integer getFlight() {
+		return flight;
+	}
+
+	public Integer getRow() {
+		return row;
+	}
+	
+	public Character getSeat() {
+		return seat;
+	}
+	
+	/**

```

Comment:
A comment beginning with `/**` is a Javadoc comment which is syntactically associated with the immediately-following class, field, or method; you should either revise this comment to describe the no-argument constructor (*not* the constructor*s* in the plural), or remove the extra `*`.